services/api/src/models/Quests.test.ts

```diff
@@ -65,6 +66,49 @@ describe('quest', () => {
         .catch(done.fail);
     });
 
+    test('return private quests (alongwith public quests) when showPrivate is set to true', (done: DoneFn) => {
+      testingDBWithState(quests)
+        .then((tdb) => {
+          return searchQuests(tdb, '', {partition: PUBLIC_PARTITION, showPrivate: true});
+        })
+        .then((results) => {
+          console.log(results);
+          expect(results.length).toEqual(2);
```

Comment:
Note that this test could use a more robust `expect`, since the length check will fail every time a new quest is added to this section. Instead, we should confirm that at least one result has partition = public, and at least one = private

Reply:
Yes, makes sense

Reply:
Looks like this still needs to be fixed?